mkstage4.sh

```diff
@@ -80,7 +93,7 @@ shift;OPTIONS="$@"
 EXCLUDES="\
 --exclude=.bash_history \
 --exclude=dev/* \
---exclude=lost+found \
+--exclude=lost+found/* \
```

Comment:
Do you want to add an option for it? `--keeplost` to keep it specific, or maybe a general `--backup` option which might also hold other backup-specific functions in the future? I think I would prefer the former, but I'll merge either.